obal/data/roles/diff_package/tasks/koji_tag.yml

```diff
@@ -1,17 +1,17 @@
 ---
 - name: Find local NEVR for package
-  rpm_nevr:
+  rpm_nvr:
     spec_file: "{{ spec_file_path }}"
     scl: "{{ tag.scl | default(omit) }}"
     dist: "{{ tag.dist | default(omit) }}"
     macros: "{{ tag.macros | default(omit) }}"
-  register: package_nevr
+  register: package_nvr
 
 - name: Verify package build exists for tag
   check_koji_build:
     tag: "{{ tag.name }}"
-    nevr: "{{ package_nevr.nevr }}"
-    package: "{{ package_nevr.name }}"
+    nevr: "{{ package_nvr.nvr }}"
```

Comment:
Should the parameter be also renamed? Or is it used as real nevr still somewhere?

Reply:
Good point, yes.